### ISSUE TITLE
Plugin channels do not want to resolve

### ISSUE THREAD

**Issue by abrooksv** [JetBrains/gradle-intellij-plugin#320]
Using the following results in failing to resolve Scala plugin
`    
plugins = ["PythonCore:2018.2.182.3684.101", "org.intellij.scala:2018.2.804@nightly"]`

```
   > Could not resolve com.jetbrains.plugins:org.intellij.scala:nightly.
     Required by:
         project :jetbrains-core
      > Could not resolve com.jetbrains.plugins:org.intellij.scala:nightly.
         > Could not get resource 'file:/Users/.../.gradle/caches/modules-2/files-2.1/com.jetbrains.intellij.idea/ideaIC/2018.2/e935ec9f9231a0789831f04f48053824c57b4bf5/ideaIC-2018.2-sources.jar/org.intellij.scala-nightly-2018.2.804.jar'.
            > Could not get file details of /Users/.../.gradle/caches/modules-2/files-2.1/com.jetbrains.intellij.idea/ideaIC/2018.2/e935ec9f9231a0789831f04f48053824c57b4bf5/ideaIC-2018.2-sources.jar/org.intellij.scala-nightly-2018.2.804.jar: could not stat file (errno 20: Not a directory)
```

Tried using version `0.3.10`, works in `0.3.7`

**Comment by zolotov** [comment]
Ouch, sorry about this, will publish 0.3.11 asap

**Comment by ingokegel** [comment]
Custom channels still don't work in 0.3.11, for example with the plugin "org.jetbrains.kotlin:1.3.0-rc-116-IJ2018.3-1@EAP-NEXT". This plugin can be resolved in 0.3.7.
